### ISSUE TITLE
Rounding mode is broken on shrec/arm

### ISSUE THREAD

**Issue by skmp** [reicast/reicast-emulator#475]
For correctness, we need to sync the SH4 rounding mode with the host. An easy repro is SOTB, the intro before the first level has broken animations if this is not done correctly.

This is done for x86, but is broken for arm / x64

**Comment by theurgus** [comment]
Heya, @skmp.
I've looked into the code today, and it seems that you may've broken it on accident. [Here's some suspicious code](https://github.com/reicast/reicast-emulator/commit/7393d337e74bc2a2b6b4384acdd66bab94395391#diff-fd1b3d5a134df96f549bfeae79e1a2d1R384). I believe you intended to comment the #define out rather than set it to 0. As the result, the (arguably) all-important aarch64 dynarec is all but crippled, delegating trivial operations to fallbacks. Please clarify your intent, or, if you're too busy, I'll look into this sometime this week, once I figure out how to build and test Android builds.

**Comment by skmp** [comment]
Yes, looks like arm64 is disabled by accident. We need to enable it and do
a full QA pass on it.

**Comment by theurgus** [comment]
Just so you know, the SOTB is only glitched while running on the dynarec, the interpreter works fine (if slow). So, the SetFloatStatusReg function works correctly on aarch64 at least.

**Comment by skmp** [comment]
@DimitrisVlachos assigning this to you for the 20.05 sprint :)
